packages/adapters/fastify/web/src/resolveOptions.ts

```diff
@@ -83,7 +83,7 @@ function isFullyQualifiedUrl(url: string) {
     // eslint-disable-next-line no-new
     new URL(url)
     return true
-  } catch (e) {
+  } catch {
```

Comment:
As of https://github.com/eslint/eslint/issues/17974 -> https://github.com/eslint/eslint/pull/18043, these are now flagged by default by `no-unused-vars`. They were unused before.